<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Backlog Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QComboBox" name="combobox_column_filter"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineedit_column_filter">
        <property name="placeholderText">
         <string>Filter für Spalten...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QComboBox" name="combobox_category_filter">
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="button_apply_filter">
        <property name="text">
         <string>Filter Anwenden</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0" rowspan="2">
     <widget class="QTableWidget" name="backlog_collection_table">
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>300</height>
       </size>
      </property>
      <property name="autoScrollMargin">
       <number>16</number>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Titel</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Beschreibung</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>Star Wars: Episode III</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>Die Rache der Sith</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>Erledigt</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QScrollArea" name="scroll_area">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>342</width>
         <height>638</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_4"/>
      </widget>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Infobox</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Neu</string>
    </property>
    <addaction name="action_new_backlog"/>
    <addaction name="action_new_category"/>
   </widget>
   <widget class="QMenu" name="menuMail">
    <property name="title">
     <string>Bearbeiten</string>
    </property>
    <addaction name="action_edit_backlog"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionHilfe"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuMail"/>
   <addaction name="menuHilfe"/>
  </widget>
  <action name="actionMail_erfassen">
   <property name="text">
    <string>Mail erfassen</string>
   </property>
  </action>
  <action name="actionMail_senden">
   <property name="text">
    <string>Mail senden</string>
   </property>
  </action>
  <action name="actionMen_bericht_sortieren">
   <property name="text">
    <string>Menübericht sortieren</string>
   </property>
  </action>
  <action name="action_new_backlog">
   <property name="text">
    <string>Neuen Backlog hinzufügen</string>
   </property>
  </action>
  <action name="action_new_category">
   <property name="text">
    <string>Neue Kategorie anlegen</string>
   </property>
  </action>
  <action name="action_edit_backlog">
   <property name="text">
    <string>-Datensatz bearbeiten</string>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="text">
    <string>Hilfe</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
